tests/gtest/test_growth.cpp

```diff
@@ -0,0 +1,59 @@
+#include "gtest/gtest.h"
+#include "population_dynamics/growth/functors/ewaa.hpp"
+
+ 
+namespace
+{
+  TEST(growth_evaluate, integer_age_input)
+  {
+// empirical weight-at-age values from the model comparison project
+// (via the Rdata object being used by the data group)
+// ewaa(a = 0): 0.0
+// ewaa(a = 1): 0.0005306555
+// ewaa(a = 2): 0.0011963283 
+
+      // not sure why this dummy test is here
+      EXPECT_EQ(1.0, 1.0);
+      
+      // create a new ewaa singleton class
+      fims::EWAAgrowth<double> ewaa1;
+      // set the ewaa values using an initializer list
+      // std::pair is a class template that provides a way to store two heterogeneous objects as a single unit
+      ewaa1.ewaa = 
+      std::map<double,double> {std::pair<double, double> (0.0,0.0),
+      std::pair<double, double> (1.0, 0.005306555),
```

Comment:
The ewaa value should start 0.0005 instead of 0.005 as it does currently to match the comment on line 12. Currently growth decreases from age 1 to age 2. Since all the values were copy/pasted in the tests, everything passes even though the growth doesn't make sense biologically. Would be nice to fix the tests, but not really necessary.